elasticsearch/src/main/java/com/amazon/opendistroforelasticsearch/sql/elasticsearch/data/utils/ElasticsearchJsonContent.java

```diff
@@ -90,7 +90,7 @@ public Boolean booleanValue() {
 
   @Override
   public boolean isNull() {
-    return value == null || value.isNull();
+    return value == null || value.isNull() || (value.isArray() && value.isEmpty());
```

Comment:
I think some of the changes included in this PR are not meant to be added to ODFE 1.13.2- the only ones we are adding are for the version bump

Reply:
This PR is to merge into `dev`, so the changes are not originally in the `odfe 1.13.2` branch. The reason it shows here is we've done some cherry-picking before on those commits, which created a new commit id. I am thinking the best way to do this, maybe I need to cherry-pick only the version bump

Reply:
That's a good point, I forgot. If we are only including the PR then it doesn't matter